### PR TITLE
USB audio fixup in user daemon instead of pulseaudio config

### PR DESCRIPTION
**Pull request by jackpot51**
This should fix thelio-major-r2 and thelio-mega-r1 audio

### CODE REVIEW

File: system76-user-daemon.desktop [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [Desktop Entry]
 Type=Application
 Encoding=UTF-8
-Name=System76 Backlight Daemon
+Name=System76 User Daemon
 Comment=Make backlight work on GNOME with NVIDIA 9-Series hardware
```
Comment (crawfxrd) [inline system76-user-daemon.desktop:5]: Should comment be updated?

**Comment by jackpot51** [comment]
To really fix up TRX40 audio, we will need a whole bunch of shit:

- https://bugzilla.kernel.org/show_bug.cgi?id=206873
- https://git.kernel.org/pub/scm/linux/kernel/git/tiwai/sound.git/log/?h=topic/usb-trx40
- https://github.com/alsa-project/alsa-ucm-conf/pull/25
- https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/292

**Comment by jackpot51** [comment]
I need regression testing from ISO 5 to this change to be sure everything is happy

**Comment by leviport** [comment]
I'll re-test in the morning after I've retrieved major-r2.

**Comment by leviport** [comment]
On Pop ISO 5, this patch broke mic input until I fully updated the system. After installing all updates, mic input, line out, line in, and S/PDIF all work as expected.

**Comment by leviport** [comment]
Ubuntu is good as well.

**Pull request merged**